src/store/store-items.js

```diff
@@ -155,7 +163,18 @@ const actions = {
         this.$router.replace("/auth")
       }
     })
-  }
+  },
+  fbUpdateListMetaData({ dispatch, getters, state }) {
+    const payload = {
+      id: state.listId,
+      updates: {
+        priceTotal: getters.priceTotal,
+        itemsTotal: getters.itemsTotal,
+        cartTotal: getters.cartTotal,
+      }
+    }
+    dispatch('lists/updateList', payload, { root: true })
+  },
```

Comment:
This takes care of updating the list store and fb with metadata from the current list. 
It could be interesting to double check if this action is called more times than actually needed.